modules/login/php/authenticatetoken.class.inc

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Implements the token authentication for LORIS.
+ *
+ * PHP version 7
+ *
+ * @category   Behavioural
+ * @package    Main
+ * @subpackage Imaging
+ * @author     Dave MacFarlane <david.macfarlane2@mcgill.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+namespace LORIS\login;
+
+/**
+ * Implements the token authentication for LORIS.
+ *
+ * @category   Behavioural
+ * @package    Main
+ * @subpackage Imaging
+ * @author     Dave MacFarlane <david.macfarlane2@mcgill.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+class AuthenticateToken extends \NDB_Form
```

Comment:
There's not a form on this page. Why not just extend NDB_Page?

Reply:
good catch